client/store/cart.js

```diff
@@ -9,7 +9,7 @@ const DELETE_SONG_FROM_CART = 'DELETE_SONG_FROM_CART'
 const SET_LOCAL_STORAGE = 'SET_LOCAL_STORAGE'
 
 //ACTION CREATORS
-export const getCart = cart => ({
+const getCart = cart => ({
```

Comment:
Id say, lets leave export here? Im not sure if it would affect my other code if we remove it. Let me know!